src/Service.GraphQLBuilder/GraphQLUtils.cs

```diff
@@ -30,6 +30,7 @@ public static class GraphQLUtils
         public const string OBJECT_TYPE_QUERY = "query";
         public const string SYSTEM_ROLE_ANONYMOUS = "anonymous";
         public const string DB_OPERATION_RESULT_TYPE = "DbOperationResult";
+        public static HashSet<DatabaseType> RELATIONAL_DB_SUPPORTING_NESTED_MUTATIONS = new() { DatabaseType.MSSQL };
```

Comment:
`DB_TYPES_SUPPORTING_NESTED_CREATE`. I feel relational can be dropped as the intent of the variable is to hold the database types that support nested mutation operations. Maybe, having operation specific data structures might make it easier if feature disparity should come up in the future

Reply:
Does cosmos not support nested mutation operations?